### ISSUE TITLE
Tasmota UI not accessible by browser

### ISSUE THREAD

**Issue by MNEWI** [arendst/Sonoff-Tasmota#2959]
Dear community, as a new user of Sonoff devices I started to flash my units (S20, basic and Pow) by Esploader and the Sonoff.bin. 
I got the AP to do the first configuration to my Network and I can see the devices in my Fritzbox.
Unfortunately the devices are not accessible by the IP adress from whatever browser neither on Win 7 or Win 10.
The only Access I get is via a Samsung Android device based on Android 7 (Android 6 was also not working).
After an update to Androif 8 the functionality disappeared.
From my homematic base the units are accessible with CuxD command "wget" for switching.

Is there any idea why there is no access from a browser based PC or other mobile devices  ???

Would very much appreciate your Feedback.

- [x] Searched the problem in issues (https://github.com/arendst/Sonoff-Tasmota/issues)
- [x] Searched the problem in the wiki (https://github.com/arendst/Sonoff-Tasmota/wiki/Troubleshooting)
- [x] Searched the problem in the forum (https://forum.digital-pig.com/)
- [x] Searched the problem in the discussion group (https://groups.google.com/d/forum/sonoffusers)
- [x] Hardware used : 
- [x] Provide the output of command ``status 0`` :
```
STATUS 0 OUTPUT HERE
```

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Frogmore42** [comment]
Without status 0, it will be really hard to provide an useful troubleshooting hints.  You need to have a serial connection to see what Tasmota thinks is happening and the version and configuration.  It is most likely a configuration problem.

**Comment by MNEWI** [comment]
Ooh, I guess I ticked by misunderstanding all possible search fields.
Sorry was my first post here...

@Frogmore42 : Enclosed please find the answer from Status 0 command.
I anonymized the Mac ID and the SSID by X's .....

20:03:52 CMD: Status 0
20:03:52 RSL: STATUS = {"Status":{"Module":8,"FriendlyName":["Soni-ZS1"],"Topic":"sonoff","ButtonTopic":"0","Power":1,"PowerOnState":3,"LedState":1,"SaveData":1,"SaveState":1,"ButtonRetain":0,"PowerRetain":0}}
20:03:52 RSL: STATUS1 = {"StatusPRM":{"Baudrate":115200,"GroupTopic":"sonoffs","OtaUrl":"http://sonoff.maddox.co.uk/tasmota/sonoff.bin","RestartReason":"Power on","Uptime":"8T11:06:36","StartupUTC":"2018-06-03T07:57:16","Sleep":0,"BootCount":28,"SaveCount":107,"SaveAddress":"F9000"}}
20:03:52 RSL: STATUS2 = {"StatusFWR":{"Version":"5.14.0","BuildDateTime":"2018-05-15T15:29:54","Boot":31,"Core":"2_3_0","SDK":"1.5.3(aec24ac9)"}}
20:03:52 RSL: STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["XXXXXXXX",""],"TelePeriod":300,"SetOption":["00008001","55818000"]}}
20:03:52 RSL: STATUS4 = {"StatusMEM":{"ProgramSize":526,"Free":476,"Heap":19,"ProgramFlashSize":1024,"FlashSize":1024,"FlashMode":3}}
20:03:52 RSL: STATUS5 = {"StatusNET":{"Hostname":"Soni-ZS1","IPAddress":"192.111.111.43","Gateway":"192.111.111.1","Subnetmask":"255.255.255.0","DNSServer":"192.111.111.1","Mac":"XX:XX:XX:XX:XX:XX","Webserver":2,"WifiConfig":2}}
20:03:52 RSL: STATUS7 = {"StatusTIM":{"UTC":"Mon Jun 11 19:03:52 2018","Local":"Mon Jun 11 20:03:52 2018","StartDST":"Sun Mar 25 02:00:00 2018","EndDST":"Sun Oct 28 03:00:00 2018","Timezone":1,"Sunrise":"04:46","Sunset":"20:53"}}
20:03:52 RSL: STATUS10 = {"StatusSNS":{"Time":"2018-06-11T20:03:52"}}
20:03:52 RSL: STATUS11 = {"StatusSTS":{"Time":"2018-06-11T20:03:52","Uptime":"8T11:06:36","Vcc":3.180,"POWER":"ON","Wifi":{"AP":1,"SSId":"XXXXXXXX","RSSI":100,"APMac":"XX:XX:XX:XX:XX:XX"}}}
20:04:28 RSL: STATE = {"Time":"2018-06-11T20:04:28","Uptime":"8T11:07:12","Vcc":3.180,"POWER":"ON","Wifi":{"AP":1,"SSId":"XXXXXXXX","RSSI":100,"APMac":"XX:XX:XX:XX:XX:XX"}}
--

Thanks for looking into it...

**Comment by Frogmore42** [comment]
It looks like you don't have a MQTT server set up, or if you do it is not configured/working.  I'll bet if you capture the log from one, you will see it rebooting regularly.  This will make it really hard to get to the web UI.

**Comment by MNEWI** [comment]
After configuring a MQTT Server (IO Broker) from where I can activate the devices it remains the same....
Sonoff S20 and basic still not accessible from a Web Browser, except my Samsung devices with Android 7 and 8.

Does anyone have more ideas ?
Thx.

**Comment by Frogmore42** [comment]
How are you trying to access them?
Are you using the name or ip address?

**Comment by MNEWI** [comment]
I am accessing by typing the IP Adress in the browser command line, like 192.168.XXX.XXX with or without http:/ as prefix or /cs as suffix....
Tried to use the name but this does not work as well.

**Comment by MNEWI** [comment]
yes you are right, as I changed the IP in the above by purpose ...
On startup after restart the following is in the console **without any change**:
_00:00:00 Project sonoff Soni-ZS2 (Topic sonoff, Fallback Soni-ZS2, GroupTopic sonoffs) Version 5.14.0-2_3_0
00:00:00 WIF: Connecting to AP1 Natascha 2,4 in mode 11N as Soni-ZS2...
00:00:09 WIF: Connected
00:00:09 DNS: Initialized
00:00:09 HTP: Web server active on Soni-ZS2.local with IP address 192.168.178.44
00:00:11 MQT: Attempting connection...
00:00:11 MQT: Connected
00:00:11 MQT: tele/sonoff/LWT = Online (retained)
00:00:11 MQT: cmnd/sonoff/POWER =
00:00:11 MQT: tele/sonoff/INFO1 = {"Module":"S20 Socket","Version":"5.14.0","FallbackTopic":"Soni-ZS2","GroupTopic":"sonoffs"}
00:00:11 MQT: tele/sonoff/INFO2 = {"WebServerMode":"Admin","Hostname":"Soni-ZS2","IPAddress":"192.168.178.44"}
00:00:11 MQT: tele/sonoff/INFO3 = {"RestartReason":"Software/System restart"}
00:00:12 MQT: stat/sonoff/RESULT = {"POWER":"OFF"}
00:00:12 MQT: stat/sonoff/POWER = OFF
20:23:48 MQT: tele/sonoff/STATE = {"Time":"2018-06-26T20:23:48","Uptime":"0T00:00:19","Vcc":3.230,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Natascha 2,4","RSSI":76,"APMac":"34:31:C4:C0:23:F3"}}_

If I ping the IP with 32 byte, there is no response (target host not accessible) although Windows says that 4 packages have been send and 4 packages received and nothing lost.
I can not access from any browser under Win7 or Win10, either Explorer, FireFox or Opera...
The only way to access the UI is via my Samsung devices (Tab and S7) with android 7 and 8 based on Samsung browser 7.0.10.47 or 7.2.10.33

**Comment by MNEWI** [comment]
Just figured out that the cause is a very strange network issue.

I am using a Fritzbox 7490 as router, having a 2,4 GHz and a 5,0 Ghz WLan activated.
The Sonoff Devices are connected with the 2,4 WLan.

**Now figured out:**
Decives connected in the 2,4 WLan have no access to the Sonoff's as mentioned above. Also my Laptop is connect on 2,4 thus no ping and no Browser access to the UI.
My Samsung devices use the 5,0 Wlan.
Now connecting my Lap and any other with the 5,0 Wlan from all Browser having access to the UI and the Ping responses correct !!!  :)

Still strange WHY this happens.....

I have now a solution, but would love to have an explanation to understand the why....

Any comment welcome !

**Comment by MNEWI** [comment]
Sorry closed accidentally....

**Comment by skibi** [comment]
Hi, i had same problem. In my case both Laptop and Sonoff device was used guest WiFi network - which separate visibility of each member of guest network.

So for the future  - check yours router setup and see if network members can discover other members.
